tests/modules/programs/mise/nushell-integration.nix

```diff
@@ -11,12 +18,7 @@
   };
 
   nmt.script = ''
-    assertFileContains home-files/.config/nushell/env.nu \
-      '
-      let mise_path = $nu.default-config-dir | path join mise.nu
-      ^mise activate nu | save $mise_path --force
-      '
-    assertFileContains home-files/.config/nushell/config.nu \
-      'use ($nu.default-config-dir | path join mise.nu)'
+    assertFileRegex home-files/.config/nushell/config.nu \
+      'use \/nix\/store\/.*-mise-nushell-config.nu'
```

Comment:
Just a tip for future, you can also use `normalizeStorePaths` to get a static store reference to avoid having to make a regex.